### DISCUSSION TITLE
AUTHORVIEWCAPABILITY in mail templates

### DISCUSSION THREAD

**Discussion by rowlap** [kohler/hotcrp#363] [Q&A] [answered]
I'm staring at the code but can't quite figure out what `AUTHORVIEWCAPABILITY` provides to the bundled mail templates.

As it's not part of the inline docs, can I ask here what's the intended purpose?
https://github.com/kohler/hotcrp/blob/216d7505ea4275d297b32cb77fdc64faa959ae32/src/pages/p_mail.php#L186

**Comment by kohler** [comment, marked as answer]
AUTHORVIEWCAPABILITY is a sharing token that allows people not logged in to the site to view a particular paper as an author would.
